doc/rst/usingchapel/chplenv.rst

```diff
@@ -767,32 +769,73 @@ CHPL_LLVM
      * ``none`` on linux32 where Chapel LLVM support is not yet implemented
      * ``bundled`` if you've already built the bundled llvm in
        `third-party/llvm`
-     *  ``system`` if a compatible system-wide installation of LLVM is detected
+     *  ``system`` if a compatible system-wide installation of LLVM and
+        clang is detected
      * ``unset`` otherwise
 
    If CHPL_LLVM is ``unset`` you will need to either add a system-wide
    installation of LLVM or set ``CHPL_LLVM`` to ``bundled`` or ``none``.
 
    See :ref:`readme-prereqs` for more information about currently
-   supported LLVM versions.
+   supported LLVM versions and how to install them. If you are having
+   trouble getting the build system to recognize your system install of
+   LLVM, try setting ``CHPL_LLVM=system`` and ``CHPL_LLVM_CONFIG`` as
```

Comment:
What should `CHPL_LLVM_CONFIG` be set to?  Maybe linking to the new section will be sufficient

Reply:
I adjusted this section a bit.